test/blackbox-tests/test-cases/melange/emit-with-runtime-deps-edge-cases.t

```diff
@@ -53,11 +53,9 @@ The runtime_dep index.txt was copied to the build folder
   hello from file
   
 
-
-
 Test depending on non-existing paths
 
-  $ mkdir another
+  $ mkdir -p another/another-output/another
```

Comment:
I had only ran the action builder eagerly because of this test, but it looks like creating its target dir in source makes the error pop up again, so it's rather an instance of #7207 and the recursive alias work that's gonna come after it.